circle/functions

```diff
@@ -120,8 +129,7 @@ get_distro() {
     fi
   done
 
-  echo "${DEFAULT_DISTRO}"
-  return 0
+  get_default_distro
```

Comment:
Can we throw an error here instead? Since you split the release script flow for apps images and base images, the latter doesn't need to call `get_distro` anymore